graphql-cli/src/main/java/io/ballerina/graphql/cmd/Constants.java

```diff
@@ -25,16 +25,23 @@ public class Constants {
 
     // GraphQL command line tool messages
     public static final String MESSAGE_FOR_MISSING_INPUT_ARGUMENT = "The input file path argument is missing. " +
-            "Please provide the path of the GraphQL config file or Ballerina service file " +
+            "Please provide the path of the GraphQL config file, Ballerina service file or GraphQL schema file " +
             "with -i or --input flag. \ne.g: bal graphql --input <GraphQL configuration file>";
-    public static final String MESSAGE_FOR_INVALID_FILE_EXTENSION = "A GraphQL configuration file " +
-            "with .yaml/.yml extension or a Ballerina service file with .bal is required to generate the artifacts. " +
-            "\nPlease provide the path of the GraphQL config file with -i or --input flag." +
+    public static final String MESSAGE_FOR_INVALID_FILE_EXTENSION =
+            "File \"%s\" is invalid. Supported files are,\nA GraphQL configuration file " +
+            "with .yaml/.yml extension, a Ballerina service file with .bal or a GraphQL schema file with .graphql is " +
+            "required as the input. \nPlease provide the path of the GraphQL config file with -i or --input flag." +
```

Comment:
Shall we go through the error messages again? It seems we can improve them further. check other places.